rust/cbindgen.toml

```diff
@@ -77,6 +77,7 @@ include = [
     "SIPState",
     "ModbusState",
     "CMark",
+    "PgsqlTransaction",
```

Comment:
I was getting compilation errors without that, I believe. I _think_ that can be because Suri core is still in C? To be honest, I don't fully understand all the connections and how things work in the background, yet. But will answer once things are clearer in my head.

Reply:
Tried removing it now: `output-json-pgsql.c` complains, because I'm using `PgsqlTransaction` there. So, that's what made me add it :P

Reply:
If you look at the DNS parser, it doesn't bother with this assignment from `void *` to `DNSTransaction` before passing to Rust, it just passes through the void pointer which works just as well.  So thats one way to remove this item from the `cbindgen` config.  Unfortunately in this context it doesn't provide any extra type safety.

Reply:
I see, thanks for the lead. Is that the preferred way to address such cases?

Reply:
> I see, thanks for the lead. Is that the preferred way to address such cases?

Yeah, I think so.  The Rust function takes a void ptr as well and does its own cast, so doing it C provides no value.

Reply:
Noted! Thanks again :)